renovate.json

```diff
@@ -8,5 +8,8 @@
     "group:recommended",
     "replacements:all",
     "workarounds:all"
-  ]
+  ],
+  "constraints": {
+    "go": "1.21"
```

Comment:
nit: pins also go to 1.21 in order to avoid https://github.com/corazawaf/coraza-caddy/pull/163